src/renderers/webxr/WebXRManager.js

```diff
@@ -708,7 +732,96 @@ class WebXRManager extends EventDispatcher {
 
 			if ( onAnimationFrameCallback ) onAnimationFrameCallback( time, frame );
 
-			xrFrame = null;
```

Comment:
@rcabanier is there some specific reason we `null` out this `xrFrame`? I'd like to be able to use this variable outside of the animation loop for creating anchors.  I didn't see a really critical reason to clear it out as it seems to represent the last known xrFrame.

Reply:
Is the XRFrame the regular WebXR frame or is it something from three?
Maybe we're nulling it out because you're not supposed to use it outside the Raf call. Since you're still in that call, it is ok to use it.

Reply:
Fairly certain this is the right thing to do:

* the entire example works using the cached value
* the interface for creating anchors is ergonomic without passing in a frame
* the controller events don't seem to have frame on them 
<img width="978" alt="Screen Shot 2022-11-01 at 1 50 54 AM" src="https://user-images.githubusercontent.com/294042/199195404-a58add00-8f23-4402-9085-f9db47f796ed.png">